src/main/scala/mesosphere/marathon/MarathonSchedulerDriver.scala

```diff
@@ -26,7 +26,7 @@ object MarathonSchedulerDriver extends StrictLogging {
       .setHostname(config.hostname())
 
     // Set the role, if provided.
-    config.mesosRole.foreach(frameworkInfoBuilder.setRole)
+    config.mesosRole.foreach(frameworkInfoBuilder.setRole: @silent)
```

Comment:
What is the deprecation warning about here? I thought it's just a different method we should call.

Reply:
`setRole`. It's been showing for a long time.

Reply:
I raised another question in the mesos channel

https://mesos.slack.com/archives/C1K2TAJ1L/p1531969087000039

Reply:
Perhaps we should just turn on multi-role. It doesn't look terribly difficult to support and we could probably get by just grabbing the first role or assuming nothing.

Reply:
I added an acceptance criteria to this ticket saying that we should remove all of these `@silent` tags when implementing multi-role support.

https://jira.mesosphere.com/browse/MARATHON-2775

Reply:
Thanks for the explanation. I thought it was a matter of just switching to `setRoles` but it seems a little more.